eth-connector/src/lib.rs

```diff
@@ -498,8 +526,14 @@ impl Withdraw for EthConnectorContract {
         let sender_id = env::predecessor_account_id();
         // Burn tokens to recipient
         self.ft.internal_withdraw(&sender_id, amount);
+
+        let fee_amount =
```

Comment:
I think it's necessary to produce logs on the amount of fee and withdrawal amount. The same thing for the deposit.

Reply:
0e6d3639563e74e3ca908bab4e429e714da404a0